lib/cog_cmd/ecs/task/list.rb

```diff
@@ -43,6 +44,8 @@ def fetch_task_definitions(client, params, next_token = nil, defs = [])
     end
 
     def process_definition_arns(arns)
+      return unless arns.compact.length > 0
```

Comment:
Ah, my bad. Good catch. It should probably return an empty list instead of `nil`.